### PR TITLE
Recommend using Yarn instead of NPM in changelog and readme

### PR DESCRIPTION
**Pull request by christer88**
TheLounge has started using Yarn, but does also support NPM. Yarn seems to in some cases solve issues people experience when using NPM, e.g. Sqlite3 installation.

I think it's good and about time to recommend using Yarn instead of NPM.

### CODE REVIEW

**Comment by jmcbee** [comment]
Are people experiencing problems installing thelounge via npm? If not, we're not really solving a problem. Besides, npm has already improved.

**Comment by MiniDigger** [comment]
Yarn is already recommended in the readme, this just fixed the changelogs for consistency.

File: scripts/changelog.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ Please refer to the commit list given above for a complete list of changes, or w
 As with all pre-releases, this version requires explicit use of the \`next\` tag to be installed:
 
 \`\`\`sh
-npm install -g thelounge@next
+[sudo] yarn global add thelounge@next
```
Comment (McInkay) [inline scripts/changelog.js]: Remove sudo, you don't need root for yarn global, and we shouldn't recommend that.

Comment (MiniDigger) [reply scripts/changelog.js]: should be changed in readme here too then https://github.com/thelounge/thelounge/blob/master/README.md#running-stable-releases-using-yarn-recommended

**Pull request merged**